src/ORM/Query.php

```diff
@@ -1299,6 +1299,38 @@ public function insert(array $columns, array $types = [])
         return parent::insert($columns, $types);
     }
 
+    /**
+     * Insert multiple rows of data in one call make use of bulk inserts.
+     *
+     * @param  array  $columns The columns to insert as ['columnName', 'columnName']
+     * @param  array  $values The values to insert as [['columnName' => 'value'], ... ] or ['value', ...]
```

Comment:
I am not sure if the numeric use case for values is common enough, as this would mean a single column, which is not really often happening, is it?

Reply:
Yes you can pass an array with just the values, those values line up with the columns.